ledger/participant-local-store/src/main/scala/com/daml/platform/localstore/CachedIdentityProviderConfigStore.scala

```diff
@@ -67,9 +85,23 @@ class CachedIdentityProviderConfigStore(
     .updateIdentityProviderConfig(update)
     .andThen(invalidateOnSuccess(update.identityProviderId))
 
+  override def getIdentityProviderConfig(issuer: String)(implicit
+      loggingContext: LoggingContext
+  ): Future[Result[IdentityProviderConfig]] =
+    idpByIssuer.get(issuer)
+
+  override def identityProviderConfigExists(id: IdentityProviderId.Id)(implicit
+      loggingContext: LoggingContext
+  ): Future[Boolean] =
+    idpCache.get(id).map {
+      case Right(_) => true
+      case _ => false
+    }
+
   private def invalidateOnSuccess(
       id: IdentityProviderId.Id
   ): PartialFunction[Try[Result[Any]], Unit] = { case Success(Right(_)) =>
     idpCache.invalidate(id)
```

Comment:
Consider dropping this cache given that: 1) there is a hard limit of the number of idps and it's very low, 2) idb cache by id would only speed it `GetIdentityProviderConfig` LAPI requests and is not used during authorizing LAPI requests

Reply:
Removed